tests/test_click.py

```diff
@@ -9,7 +9,7 @@
 
 @sync
 async def test_click_button(isolated_page, server):
-    """shoud click the button"""
+    """should click the button"""
```

Comment:
In the future, try to keep PRs specific in scope, that way it's easier to review. I wouldn't bother reverting it though this time because it's such a small change 😜

Reply:
Copy that ))